tensorflow/contrib/cmake/tf_core_kernels.cmake

```diff
@@ -92,6 +92,8 @@ if(tensorflow_BUILD_CONTRIB_KERNELS)
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_manager.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_ops.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/ops/nccl_ops.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/kernels/periodic_resample_op.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/ops/array_ops.cc"
```

Comment:
Can you remove these lines, and the corresponding change on L166-167? These are basically adding two filenames to the list, then removing them again :).